Google.Benchmark.nuspec

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Google.Benchmark</id>
+        <version>1.0.0</version>
+        <authors>https://github.com/google/benchmark/blob/master/AUTHORS</authors>

```

Comment:
`${GIT_VERSION}` should already have this for cmake.. but that would require you to generate the nuspec from cmake instead of checking it in. i don't know how that works.